compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5223,13 +5223,12 @@ bool directMemoryStoreHelper(TR::CodeGenerator* cg, TR::Node* storeNode)
    {
    if (!cg->getConditionalMovesEvaluationMode())
       {
-      if (!storeNode->getOpCode().isReverseLoadOrStore()
-              && storeNode->getType().isIntegral()
+      if (!storeNode->getType().isIntegral()
```

Comment:
Is this ! correct? Looks like it used to be !isReverseLoadOrStore && isIntegral but now !isIntegral?

Reply:
Oops, you're right that this is incorrect. Guess I missed the `!` when removing that part of the condition. Should be fixed now.